src/core/types/asset_definition.rs

```diff
@@ -17,42 +17,34 @@ use crate::{
 /// onboarded and verified.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-pub struct AssetDefinitionV2 {
+pub struct AssetDefinitionV3 {
     /// The unique name of the asset associated with the definition.
     pub asset_type: String,
-    /// A link to a scope specification that defines this asset type.
-    pub scope_spec_address: String,
     /// Individual verifier definitions.  There can be many verifiers for a single asset type.
     pub verifiers: Vec<VerifierDetailV2>,
     /// Indicates whether or not the asset definition is enabled for use in the contract.  If disabled,
     /// requests to onboard assets of this type will be rejected.
     pub enabled: bool,
```

Comment:
We can add it in a follow-up if you'd rather this just go as-is, but it would be cool to sneak that `display_name: Option<String>` in here if you get a chance

Reply:
oh yeah, I'll drop that in, just slipped my mind

Reply:
done and done push'd